packages/aws-cdk/lib/cli/io-host/cli-io-host.ts

```diff
@@ -180,7 +180,7 @@ export class CliIoHost implements IIoHost {
       });
     }
     // TODO: uncomment this at launch
-    // if (canCollectTelemetry(context)) {
+    // if (canCollectTelemetry(args, context)) {
```

Comment:
`canCollectTelemetry` is not in use in the pre-launch phase where we are going to launch telemetry to local file (since the opt-in is the `--telemetry-file` itself, irregardless of our opt-out options). it is in use in `cdk cli-telemetry --status`, however.